imports/api/resolvers.js

```diff
@@ -0,0 +1,26 @@
+import { Lists } from './lists/lists';
+
+export default {
+  RootQuery: {
+    lists(_, { mine = false }, context) {
+      if (mine) {
+        // XXX: is this the correct way to do auth?
```

Comment:
I guess I meant returning an empty array rather than throwing some kind of exception. Is that what you meant?